holoviews/selection.py

```diff
@@ -325,6 +325,24 @@ def selection_param(self, data):
         self._datasets.append((pipe, data, raw))
         return pipe.param.data
 
+    def filter(self, data):
+        """
+        Filters the provided data based on the current state of the
+        current selection expression.
+
+        Args:
+            data: A Dataset type or data which can be cast to a Dataset
+
+        Returns:
+            The filtered data
+        """
+        if self.selection_expr is None:
+            return data
+        if not isinstance(data, Dataset):
+            raw = True
```

Comment:
What does this line achieve?

Reply:
Nothing, good catch, was rushed and copy pasted.

Reply:
```suggestion
```